javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,167 @@
+// Flight booking full name function =============================================================================
+
+const getFullName = (firstName, lastName) => `${firstName} ${lastName}`;
+
+const fullName1 = getFullName('Rostyslav', 'Zhyvitskyi');
+const fullName2 = getFullName('Benjamin', 'Hughes');
+
+console.log(fullName1, fullName2);
+
+// Formal full name
+
+const getFullNameFormal = (firstName, lastName, sex, useFormalName) => {
+  if (firstName === '' || lastName === '') {
+    throw Error('Please make sure you have entered your first and last name');
+  }
+
+  if (useFormalName) {
+    return sex === 'male'
+      ? `Mr. ${firstName} ${lastName}`
+      : `Mrs. ${firstName} ${lastName}`;
+  } else {
+    return `${firstName} ${lastName}`;
+  }
+};
+
+// Event application ===============================================================================================
+
+const weekDays = [
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+  'Sunday',
+];
+
+const getEventWeekday = (remainingDays) => {
+  const today = new Date().getDay();
+
+  let dayNumber = today + remainingDays;
+
+  if (dayNumber % 7 === 0) {
+    dayNumber = today + (remainingDays % 7);
+  } else {
+    dayNumber %= 7;
+  }
+
+  const weekDay = weekDays[dayNumber - 1];
+
+  console.log(`Party is going to be on ${weekDay} `);
+};
+
+getEventWeekday(2);
+
+// Weather wear ===================================================================================================
+
+const getClothesForTemperature = (temperature) => {
+  if (temperature <= 0) {
+    return `it's very cold outside, please put a warm jacket on you`;
+  } else if (temperature > 0 && temperature <= 10) {
+    return `it's cold outside, please put a jacket on you`;
+  } else if (temperature > 10 && temperature <= 20) {
+    return `it's not so cold outside, but don't forget to put a sweater on you`;
+  } else {
+    return `it's a good weather, just put a T-shirt on you`;
+  }
+};
+
+const clothesToWear = getClothesForTemperature(20);
+
+console.log(clothesToWear);
+
+// Student manager ===============================================================================================
+
+const class22Students = [];
+
+function addStudentToClass(studentName) {
+  if (studentName === '') {
```

Comment:
`addStudentToClass();` would work in this case, and it shouldn't

Reply:
Fixed😊